FR.json

```diff
@@ -0,0 +1,13 @@
+{
+    "loadButtonLabel": "Cliquez pour\ncharger\nle panorama",
+    "loadingLabel": "Chargement...",
+	"bylineLabel": "de %s",    
```

Comment:
shouldnt this be "par %s" because its not "of %s" but its "by %s" ?

this refers to a string that shows the author of the panorama e.g. "mona lisa - by leonardo da vinci".

Reply:
I hesitated, both are right, you could say "de" or "par".
"by" is frequently translated as "par" but in French we most often say "de" when talking about the author of something.

Reply:
Interesting, I didnt know that.

Reply:
Just realize there was already a pull request for French when I opened mine. Please check it as well, but it's pretty close to this one. Regarding this review, I'd suggest "par" instead of "de".

Reply:
You're right, when talking about a picture "par" should be used ("de" is better for art pieces)